src/solver/conv_asm_3x3u.cpp

```diff
@@ -190,12 +190,14 @@ bool ConvAsm3x3U::IsApplicable(const ConvolutionContext& params) const
         return false;
     }
 
-    constexpr auto GIB     = static_cast<int64_t>(1024) * 1024 * 1024;
```

Comment:
What is wrong with GIB?

Reply:
Everything is okay, i tried to make it look like in other solvers, fixed in 7ddf090